public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionCompleteProgressBar.js

```diff
@@ -20,6 +20,7 @@ function ModalMissionCompleteProgressBar () {
     var horizontalBarPreviousContribution = gBarChart.selectAll("rect")
         .data([0])
         .enter().append("rect")
+        .attr('id', 'blueBar')
```

Comment:
This is my taste, but I prefer separating words with `-` for HTML/CSS id/class tags (just so I don't confuse them with JavaScript variables). So could you change it (and other ids) accordingly? Also, it would be great if you can use more descriptive name, like `mission-completion-blue-bar` instead of just `blueBar`.